language/vm/vm_genesis/src/lib.rs

```diff
@@ -206,6 +206,43 @@ pub fn encode_transfer_script(recipient: &AccountAddress, amount: u64) -> Script
     )
 }
 
+/// Encode a program transferring `amount` coins from `sender` to `recipient` but padd the output
+/// bytecode with unreachable instructions.
+#[cfg(any(test, feature = "testing"))]
+pub fn encode_transfer_script_with_padding(
```

Comment:
Why does this code belong in the same file as the genesis generation code? It seems unrelated.

Reply:
It was where all the other transactions were. Lemme clean this up. (This should also help us clean our dependency story to make client get rid of the VM btw)